omero/sysadmins/server-upgrade.rst

```diff
@@ -235,6 +246,14 @@ intermediate major release.
    connected to the database using **db_user** before running the script. See
    :forum:`this forum thread <viewtopic.php?f=5&t=7778>` for more information.
 
+.. warning::
+
+   The :file:`sql/psql/OMERO5.4__0/OMERO5.3__1.sql` script referenced by
+   the above :program:`psql` command assumes upgrade from OMERO 5.3.4.
+   If you are instead currently running OMERO 5.3.0 to 5.3.3 then you
```

Comment:
This "5.3.0 to 5.3.3" confused me because on first glance I parsed it as running the upgrade from 5.3.0 to 5.3.3 - could you rephrase to "running OMERO 5.3.3 or an earlier 5.3.x version" please?